src/Arrays/MPIStateArrays.jl

```diff
@@ -169,6 +174,32 @@ function Base.getproperty(Q::MPIStateArray{FT, V}, sym::Symbol) where {FT, V}
     end
 end
 
+"""
+    getstateview(Q, fieldname)
+
+Returns of real element view of the state `fieldname`. The MPI state array `Q`
+must have been generated in a `Vars`-aware fashion. This is mainly to allow accessing fields which line inside of the nested `Vars` state.
+
+The state specified by 'fieldname' can be a 'String', 'Symbol', or 'Expr'
+
+# Examples
+```julia-repl
+julia> S = @vars(x::Float64, y::@vars(α::Float64, β::SVector{3, Float64}))
+julia> Q = MPIStateArray{Float64, S}(MPI.COMM_WORLD, Array, 2, 5, 3)
+julia> β = MPIStateArrays.getstateview(Q, "y.β")
+julia> β = MPIStateArrays.getstateview(Q, :(y.β))
+julia> x = MPIStateArrays.getstateview(Q, :x)
+```
+"""
+
```

Comment:
```suggestion
```

Reply:
(Documenter doesn't allow a space between the doc string and function)

Reply:
Thanks for the catch!